fraud/sync.go

```diff
@@ -0,0 +1,84 @@
+package fraud
+
+import (
+	"context"
+
+	"github.com/libp2p/go-libp2p-core/event"
+	"github.com/libp2p/go-libp2p-core/peer"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+
+	pb "github.com/celestiaorg/celestia-node/fraud/pb"
+)
+
+func (f *ProofService) syncFraudProofs(ctx context.Context) {
+	log.Debug("start fetching Fraud Proofs")
+	sub, err := f.host.EventBus().Subscribe(&event.EvtPeerIdentificationCompleted{})
+	if err != nil {
+		log.Error(err)
```

Comment:
should probably log out a warning here too that node won't sync fraud proofs and to prompt user to restart or something

Reply:
I think we should leave `log.Error` here

Reply:
I would even panic there lol.

Reply:
In practice, the error there is impossible

Reply:
let's leave log.Error

Reply:
ok